.github/workflows/install-mdbook/action.yml

```diff
@@ -8,15 +8,15 @@ runs:
     # The --locked flag is important for reproducible builds. It also
     # avoids breakage due to skews between mdbook and mdbook-svgbob.
     - name: Install mdbook
-      run: cargo install mdbook --locked --version 0.4.35
+      run: cargo install mdbook --locked --version 0.4.36
```

Comment:
Is there a corresponding change needed somewhere else to use this version when building locally?

Reply:
We tell people to install the latest version and that's normally good enough.